doc/source/tutorials/vector_api_tut.rst

```diff
@@ -22,28 +22,26 @@ Initially it is necessary to register all the format drivers that are desired.
 This is normally accomplished by calling :cpp:func:`GDALAllRegister` which registers
 all format drivers built into GDAL/OGR.
 
-In C++ :
+.. tabs::
```

Comment:
It's odd how the earlier steps in this tutorial (and the writing tutorial) don't include Python. (That precedes your changes, I know.) Do you want to add Python tabs where applicable?

Reply:
I've added the Python tabs and code from the sample scripts to match the C and C++ examples